rest/endpoints/user_recordings/profile_recordings.yml

```
@@ -381,6 +381,36 @@ paths:
                     type: array
                     items:
                       $ref: "https://raw.githubusercontent.com/booxmedialtd/api-documentation/master/rest/snippets/General/ForbiddenEntity.yml#/ForbiddenError"
+  # ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+
+  /most_popular/{period}:
+    get:
+      description: |
+        * **Available since:** BPLAT-12223
```

please include the release number
```suggestion
        * **Available since:** 2020-01 (BPLAT-12223)
```